### PR TITLE
The newsweek.com clone

### PR DESCRIPTION
**Pull request by Abdoulaye-Thespy**
In this project, I built a responsive clone of the web page of the newsweek.com.
The project is meant to be an avenue for us to practice building responsive web designs.using boostrap
It was built with html and css and boostrap.

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi @Abdoulaye-Thespy  👋

This project looks awesome. You've done quite well on this.🚀

However, we are just one step away from a thumbs-up, kindly fix this issue:

- [ ] Please on **tablet screen sizes** add more left and right margin to the sections that come below the `IN THE MAGAZINE` section. 

**THIS IS YOURS**👇
ice_screenshot_20200423-191302

**THIS IS WHAT IS EXPECTED**👇
ice_screenshot_20200423-191317

Kindly fix these issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by men32z — CHANGES_REQUESTED** [review]
Hey @Abdoulaye-Thespy !
## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

### all changes are REQUIRED

#### take a look in the image, yours is the top one
sc1
in wide screens
- [ ] add the white background in the main section, you have currently just the gray one.
- [ ] add padding to subscribe button

#### take a look in the image, yours is the left one
sc2
in small screens 
- [ ] fix the size of the navbar in small screen sizes.
- [ ] use the right font size and bold for the titles like "TOP STORY"
- [ ] make sure images don't get distorted on resizing the window.

#### take a look in the image, yours is the left one
sc3
- [ ] make the changes on small screens for the "the starting 5" section it changes.
- [ ] remove in small screens the "|" separators in the footer menu.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! :hammer:

--Luis Preza

## Status Request changes

**Comment by sumancrest0001** [comment]
**Good job @Abdoulaye-Thespy,** :clap:  :clap: 
You have fixed most of the issues addressed by the previous TSEs but some of them are still there. We need to fix the following issues to get approval for this project. 
**Note: All the changes are compulsory.**

We have to fix the alignment of nav items on the large screen. Let's align the subscribe button at the bottom.
Currently :point_down: 
Screenshot from 2020-04-24 14-10-35

It should be :point_down: 
Screenshot from 2020-04-24 14-10-48

**Comment by sumancrest0001** [comment]
Please fix the sign-in column on the medium screen. Currently, it is covering the whole screen but It should be centered and must not cover the whole screen. See the difference.
Screenshot from 2020-04-24 14-01-41
on the actual page :point_down: 
Screenshot from 2020-04-24 14-02-03

**Comment by sumancrest0001** [comment]
We have to fix the alignment of the title and image on a small screen. 
currently :point_down: 
Screenshot from 2020-04-24 13-47-33
it should be :point_down: :point_down: 
Screenshot from 2020-04-24 13-48-32

**Comment by sumancrest0001** [comment]
The bottom Newsweek logo section is still not fixed for small screen devices. Please solve the alignment issue.
currently :point_down: :point_down: 
Screenshot from 2020-04-24 13-51-20

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+}
+
+.sus {
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  padding-top: 0.1rem;
+  padding-bottom: 0.1rem;
+  margin-bottom: 1rem;
+}
+
```
Comment (sumancrest0001) [inline styles.css]: We can define margin and padding in one line. For example, `margin: top right bottom left`. We should follow the DRY principle [reference](https://www.w3schools.com/cssref/pr_margin.asp)
Please try to match font-size and font-weight if not font-family of the original site as it is a clone project.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
Please make the above-requested changes and submit for another review.
**If you have any questions or need help, do not hesitate to contact me on slack (Suman Shrestha)**the 
All the best :+1:

**Comment by uryelah** [comment]
Hello **Abdoulaye Razak**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

Great job fixing the last required changes.

------
## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

----

## Optional Changes (not required but will make your project more professional)

- [ ] Your footer logo has a background-color. This is not good as it looks weird if the footer background color is different, in this case, white against grey. 
Make sure you ALWAYS use svg images for logos or png images with a transparent background. 
[To learn more about svg read this article. ](https://svgontheweb.com/)
Tip, you can get the logo svg directly from the original site if you inspect it.


**Keep being an awesome dev and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**